spec/requests/api/v1/npq_participants_spec.rb

```diff
@@ -77,10 +77,6 @@
         end
 
         context "filtering" do
-          before do
-            create_list(:npq_application, 2, npq_lead_provider: npq_lead_provider, updated_at: 10.days.ago, school_urn: "123456", npq_course: npq_course)
```

Comment:
I don't quite understand why this is no longer needed? 
is it because now we filter on `users.updated_at`?